README.md

```diff
@@ -122,13 +122,11 @@ make installworld DESTDIR=/iocage/releases/custom/root
 ioc fetch -r custom -b
 ```
 
-## Development
+## Documentation
 
-### Install Development Dependencies
+The [API Reference (html)](https://iocage.github.io/libiocage) documenting all public interfaces of libiocage is updated with every release. The information found in the reference is compiled from Python docstrings and MyPy typings using Sphinx.
 
-```sh
-make install-dev
-```
```

Comment:
is there a reason this is nuked?

Reply:
It is moved in the readme. In my opinion it makes more sense to show it related to the `make check` command.